build.gradle.kts

```diff
@@ -72,7 +76,7 @@ description = "OldCombatMechanics"
 java {
     toolchain {
         // At least 17 required for MC 1.19 for ingametesting
-        languageVersion.set(JavaLanguageVersion.of(8))
+        languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
We can't bump the minimum java version without breaking compatibility with people using older versions e.g. 1.12

Reply:
It seems that using folia-api must use java17 to compile. I don’t know how to solve it for the time being.

Reply:
Spigot up to 1.20.3-R0.1 uses Java 8. Are there older versions of the Folia API?

Reply:
The minimum folia-api version is 1.19.4

Reply:
I have modified it to build with java8